pkg/catalog-cd/catalog/testdata/contract.simple.yaml

```diff
@@ -0,0 +1,5 @@
+tasks:
+- name: git-clone
+  file: task-git-0.0.1.yaml
+  bundle: ghcr.io/openshift-pipelines/task-git:0.0.1-bundle
```

Comment:
Nice one! Which makes me thing about using the image SBOM(s) to register the author, supply chain attestation, etc. Using the bundle image might be easier than trying to do that on "plain yaml" files.

Reply:
Yep. Right now it's *ignored* but the goal is to, over time, use this as **the** way to gather most informations. But for this we need to streamline tekton bundle, make sure they are marked as stable, and easy to generate (which.. is half the case today).